src/dials/util/image_viewer/slip_viewer/frame.py

```diff
@@ -203,22 +203,26 @@ def handle_position_event(self, event):
             )
 
             posn_str = "Picture:  fast={:.3f} / slow={:.3f} pixels.".format(
-                fast_picture,
-                slow_picture,
+                fast_picture + 0.5,
+                slow_picture + 0.5,
             )
             coords = self.pyslip.tiles.get_flex_pixel_coordinates(lon, lat)
             if len(coords) >= 2:
                 if len(coords) == 3:
                     readout = int(round(coords[2]))
                 else:
                     readout = -1
-
+                # the dials convention is that the center of the pixel is 0.5,0.5, so the extent
+                # of the 0,0 pixel is from 0.0,0.0 to 1.0,1.0 in pixel space.  Here we are
+                # get_flex_pixel_coordinates defines the center of the pixel as 0.0,0.0, so we
```

Comment:
And here is where we have the inconsistency betwixt what we think in DIALS and what the image viewer says.

We should probably add a comment to the implementation of `get_flex_pixel_coordinates` that it defines the 0,0 postion as the centre